int32 ts
float64 latitude
float64 longitude
float32 altitude
float32 height
float32 uncertain
